README.md

```diff
@@ -44,11 +44,17 @@ You need to install autoconf, automake and libtool:
 
 	brew install autoconf automake libtool
 	
+In case OpenSSL is not installed:
+    
+	brew install openssl
+	cd /usr/local/include
+	ln -s ../opt/openssl/include/openssl .
```

Comment:
I also would recommend not using soft link. I tailor my installations of homebrew and I would not want to have to investigate a solution that forces to edit files in case the default solution would error. What was wrong with specifying the openssl path for configure?

Also, how come we need OpenSSL everywhere now?